pkg/multicluster/utils.go

```diff
@@ -35,22 +34,19 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
 
+	multiclusterpkg "github.com/kubevela/pkg/multicluster"
+	prismclusterv1alpha1 "github.com/kubevela/prism/pkg/apis/cluster/v1alpha1"
 	"github.com/oam-dev/cluster-gateway/pkg/apis/cluster/v1alpha1"
 	clustercommon "github.com/oam-dev/cluster-gateway/pkg/common"
 
-	velatypes "github.com/oam-dev/kubevela/apis/types"
 	"github.com/oam-dev/kubevela/pkg/oam"
 	"github.com/oam-dev/kubevela/pkg/utils/common"
 	errors3 "github.com/oam-dev/kubevela/pkg/utils/errors"
 )
 
-type contextKey string
-
 const (
-	// ClusterContextKey is the name of cluster using in client http context
-	ClusterContextKey = contextKey("ClusterName")
 	// ClusterLocalName specifies the local cluster
-	ClusterLocalName = velatypes.ClusterLocalName
+	ClusterLocalName = multiclusterpkg.Local
```

Comment:
It seems sometimes we import the package as "pkgXXX" and sometimes "xxxpkg" and sometimes "velaxxx". Can you explain the rule?

Reply:
I want to comment and hit "approve" unexpectedly

Reply:
velaxxx is okay. I think we do not need xxxvela.

Reply:
The package alias here is not a recommended way. I prefer to use multicluster.Local here but since we have a internal multicluster package and a external multicluster package, here I use multiclusterpkg to workaround, but pkgmulticluster is also okay.

Reply:
refactored.